tests/lib/rook/upstream.py

```diff
@@ -47,15 +47,16 @@ def build(self):
 
         self.get_golang()
         logger.info("Compiling rook...")
-        execute(command="make -j BUILD_REGISTRY='rook-build' IMAGES='ceph'",
+        cmd = f"make --directory {self.build_dir} "\
+              f"-j BUILD_REGISTRY='rook-build' IMAGES='ceph'"
+        execute(command=cmd,
                 env={"PATH": f"{self.workspace.bin_dir}/go/bin:"
                              f"{os.environ['PATH']}",
                      "TMPDIR": self.workspace.tmp_dir,
                      "GOCACHE": self.workspace.tmp_dir,
                      "GOPATH": self.workspace.build_dir},
-                log_stderr=False,
-                logger_name=(
-                    "make -j BUILD_REGISTRY='rook-build' IMAGES='ceph'"))
+                log_stderr=True,
```

Comment:
I think this was False for a reason. I think the go building is quite verbose, but if you can confirm the output is manageable then lets keep it True.